packages/sdk-react/src/core/queries/useMe.ts

```diff
@@ -29,3 +48,12 @@ export const useMyEntity = () => {
     isUSEntity,
   };
 };
+
+export const useMyEntityName = () => {
+  const { data: myEntity } = useMyEntity();
+
+  if (isOrganizationEntity(myEntity)) return myEntity.organization.legal_name;
+  else if (isIndividualEntity(myEntity))
+    return myEntity.individual.first_name + ' ' + myEntity.individual.last_name;
+  return '';
+};
```

Comment:
```suggestion
export const useMyEntityName = () => {
  const { data: myEntity } = useMyEntity();

  if (!myEntity) return '';

  if (isOrganizationEntity(myEntity)) {
    return myEntity.organization.legal_name;
  }

  if (isIndividualEntity(myEntity)) {
    const { first_name, last_name } = myEntity.individual;
    return `${first_name} ${last_name}`;
  }

  return '';
};
```

- Added null check: if (!myEntity) return '' prevents errors if myEntity is undefined or null.
- Improved readability: Destructuring const { first_name, last_name } makes the code cleaner when accessing multiple values.
- Used template literals: Replaced string concatenation with template literals for simpler and more modern syntax.